src/v/raft/vote_stm.cc

```diff
@@ -479,4 +479,13 @@ ss::future<> vote_stm::self_vote() {
     auto m = _replies.find(_ptr->self());
     m->second.set_value(reply);
 }
+
+void vote_stm::lose() {
```

Comment:
nit: a more descriptive name would be nicer

Reply:
+1

Reply:
Changed to `lose_election`, do you think it's better? (I'm not quite sure what else we can lose in a class whose purpose is to run an election)

Also if we're after naming things right I'd rename half of the `vote` to `election` to clarify whether we arrange an election or merely vote in an election already organised. Say, `consensus::vote()` vs `vote_stm::vote` confused me, as the latter runs an election, not just votes really. How conservative are we regarding existing names? Is it okay to rename things if I touch the code around them?